common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -39,7 +39,7 @@ const EmptyTile = styled.div`
 `;
 
 export default function Dashboard() {
-  const { isActive: isDevToolsActive, displayRecentTransactionList } = useDevTools();
```

Comment:
Please remove it from DevTools as well.
It's good to avoid leaving dead code lying around.